pkg/memtier/policy_age.go

```diff
@@ -312,13 +312,20 @@ func (p *PolicyAge) Start() error {
 	if p.tracker == nil {
 		return fmt.Errorf("missing tracker")
 	}
-	p.tracker.Start()
+	if err := p.tracker.Start(); err != nil {
+		return fmt.Errorf("tracker start error: %w", err)
+	}
 	p.cgLoop = make(chan interface{})
```

Comment:
With this change it is possible to create p.cgLoop without never entering p.loop(). This causes the problem that next p.Start() will immediately fail with "already started" even though the policy is  not started.

Maybe we could postpone creating p.cgLoop just before starting the p.loop() goroutine?